src/renderers/webgl/WebGLState.js

```diff
@@ -599,7 +599,6 @@ function WebGLState( gl, extensions, paramThreeToGL ) {
 
 			}
 
-			currentBlending = blending;
 			currentPremultipledAlpha = premultipliedAlpha;
```

Comment:
Okay, if @Mugen87 agrees, I will do it using a <code>switch</code> statement and then create a new pull request.